subsys/bluetooth/controller/ll_sw/ull_adv_types.h

```diff
@@ -4,6 +4,8 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+ #define INVALID_ADV_HANDLE 0xFF
+
```

Comment:
We already have `BT_HCI_ADV_HANDLE_INVALID` which we can use

Reply:
My bad, I'll fix that.